README.md

```diff
@@ -344,10 +344,12 @@ If this option is turned to true then meta objects will be parsed and will be se
 
 A timestamp in ms, must be within one day otherwise it will be dropped and Date.now() will be used in its place.
 
-### flushAll()
+### flushAll(callback)
```

Comment:
If this is a new feature, then please also add that to `CHANGELOG.md`

Reply:
We'll do before releasing